packages/redwood/src/index.ts

```diff
@@ -119,9 +120,10 @@ export async function build({
       }),
     };
 
-    dependencies.forEach(fsPath => {
-      lambdaFiles[relative(workPath, fsPath)] = new FileFsRef({ fsPath });
-    });
+    for (const fsPath of dependencies) {
+      const { mode } = statSync(fsPath);
+      lambdaFiles[relative(workPath, fsPath)] = new FileFsRef({ fsPath, mode });
```

Comment:
```suggestion
      lambdaFiles[relative(workPath, fsPath)] = await FileFsRef.fromFsPath({ fsPath });
```

Reply:
Aha, I was wondering why the mode wasn't automatically set. This explains it 👍